src/components/AssetSwiper/AssetSwiper.js

```diff
@@ -0,0 +1,56 @@
+// Import Swiper React components
+import React, { useState, useEffect } from "react";
+import { Typography, IconButton } from "@material-ui/core";
+import { Swiper, SwiperSlide } from "swiper/react";
+import AddIcon from "@material-ui/icons/Add";
+import CircularProgress from "@material-ui/core/CircularProgress";
+// Import Swiper styles
+import "swiper/swiper.scss";
+import AssetCard from "../AssetCard/AssetCard";
+import { Redirect } from "react-router-dom";
+import { useStyles } from "./Comps/AssetSwiper.style";
+import AssetSwiperMobile from "./Comps/AssetSwiperMobile";
+export default ({ list, title, profileSwiper, spinner }) => {
```

Comment:
spinner is a bad name for loading state, what is loading? call it isLoading / isLoadingList